genesyscloud/conversations_messaging_integrations_open/genesyscloud_conversations_messaging_integrations_open_proxy.go

```diff
@@ -0,0 +1,173 @@
+package conversations_messaging_integrations_open
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/mypurecloud/platform-client-sdk-go/v143/platformclientv2"
```

Comment:
I think we are on a newer version now. You might want to merge the latest version of dev into this branch and update this